css/custom.css

```diff
@@ -92,3 +92,8 @@
 .background-image {
   width: 300px;
 }
+@media all and (max-width: 767px) {
+  .footer .menu {
+    width: 100%;
+  }
+}
```

Comment:
can you please put a new line at end of the file?

Reply:
Ok sure.